lib/bloc/heroes_bloc.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:marvel/bloc/heroes_event.dart';
+import 'package:marvel/bloc/heroes_state.dart';
+import 'package:marvel/data/model/character.dart';
+import 'package:marvel/data/repository/marvel_repository.dart';
+
+class HeroesBloc extends Bloc<HeroesEvent, HeroesState> {
+  final MarvelRepository marvelRepository;
+
+  HeroesBloc(this.marvelRepository)
+      : super(const HeroesState(loading: true)) {
+    on<HeroesEvent>(
+      (event, emit) async {
+        emit(state.copyWith(loading: true));
+        try {
+          final List<Character> _loadedCharacterList =
```

Comment:
```suggestion
          final List<Character> characters =
```

Reply:
1) Names should be short and clear.
2) Underscore is for private fields.

Reply:
done